packages/xo-server/config.toml

```diff
@@ -94,3 +94,6 @@ timeout = 600e3
 
 # see https:#github.com/vatesfr/xen-orchestra/issues/3419
 # useSudo = false
+
+[xapiOptions]
+maxUncoalescedVdis = 1
```

Comment:
@olivierlambert The current value is `0`, and I propose to increase this number `1`, do you want to increase it further?

Reply:
1 is fine :+1: